install/Makefile

```diff
@@ -59,7 +29,7 @@ crd_install_helm3:
 	yq write -i ../helm-charts/helm3/strimzi-kafka-operator/crds/048-Crd-kafkamirrormaker2.yaml metadata.labels.component "kafkamirrormaker2.kafka.strimzi.io-crd"
 	yq write -i ../helm-charts/helm3/strimzi-kafka-operator/crds/049-Crd-kafkarebalance.yaml metadata.labels.component "kafkarebalances.kafka.strimzi.io-crd"
 
-crd_install: crd_install_helm2 crd_install_helm3
+crd_install: crd_install_helm3
```

Comment:
at this point I think we could even renaming from `crd_install_helm3` to just `crd_install_helm`

Reply:
I did think about doing that but I guess at some point there will be a helm 4 and we may need to run both again. If we leave it like this there is less to change. WDYT?

Reply:
Makes sense. I was wondering we were going to support just one Helm version from now on. If not, then it's fine helm3 ;-)

Reply:
Well, I guess there will be a long time until Helm4. So I do not think that is something what we will add in next months. (at the same time, I do not have problem to keep the target called helm3)